src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,73 @@
+package duke.command;
+
+import duke.Parser;
+import duke.Ui;
+import duke.Storage;
+import duke.task.TaskList;
+import duke.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+/**
+ * Represents command that is specific to the add command.
+ */
+public class AddCommand extends Command {
+
+    private String input;
+    private InputCommand inputCommand;
+
+    public AddCommand(String input, InputCommand inputCommand) {
+        this.exit = false;
```

Comment:
To make this variable sounds more boolean, you might want to consider renaming this to isExit.